app/Models/EntryDAO.php

```diff
@@ -3,11 +3,11 @@
 class FreshRSS_EntryDAO extends Minz_ModelPdo implements FreshRSS_Searchable {
 
 	public function isCompressed() {
-		return parent::$sharedDbType === 'mysql';
+		return true;
 	}
 
 	public function hasNativeHex() {
-		return parent::$sharedDbType !== 'sqlite';
+		return true;
```

Comment:
It is part of a series of change to avoid the static properties to allow working with more than one user or database at a time. Here, I use inheritance to achieve that (the boolean is different for e.g. MySQL and SQLite)

Reply:
This function specifically is to know whether the database can do the conversion between binary and hexadecimal itself (better) or whether we need a PHP fallback (for SQLite)